jekyll/_cci2/use-the-cli-and-vcs-for-config-policy-management.adoc

```diff
@@ -20,19 +20,36 @@ Use the CircleCI CLI to manage your organization's policies programmatically. Th
 The following sub-commands are currently supported within the CLI for configuration policy management:
 
 * `diff` - shows the difference between local and remote policy bundles
-* `push` - pushes the policy bundle (activate policy bundle)
 * `fetch` - fetches the policy bundle (or one policy, based on name) from remote
-
+* `push` - pushes the policy bundle (activate policy bundle)
++
 For example, running the following command activates a policy bundle stored at `./policy_bundle_dir_path`:
-
++
 [source,shell]
 ----
 circleci policy push ./policy_bundle_dir_path --owner-id <your-organization-ID>
 ----
+*  `settings` - used to modify config policy management settings as required.
```

Comment:
@sagar-connect I added the info about the settings subcommand up here with other sub commands, I wonder if we can add a little example for `diff` and `push` too for consistency?

Reply:
This will make our 'quick start' section a bit too long. Hope you agree